Wikipedia/Code/Controllers/ArticleDescriptionControlling.swift

```diff
@@ -14,8 +21,16 @@ protocol ArticleDescriptionControlling {
     func currentDescription(completion: @escaping (String?) -> Void)
     func errorTextFromError(_ error: Error) -> String
     func learnMoreViewControllerWithTheme(_ theme: Theme) -> UIViewController?
+    var descriptionMaxLength: Int { get }
```

Comment:
Not a bug, but not sure you need this line - I think you can get away with just having the variable int he protocol extension if you want. This is definitely a personal style preference, please feel very free to disagree.

Reply:
@mcleinman yep agreed, I don't see the reason to keep it here if we're not overriding/swapping out the implementation anywhere.